# CRS
Code and scripts used in manuscript 'Immune-Epithelial Dynamics and Tissue Remodeling in Chronically Inflamed Nasal Epithelium via Multi-scaled Transcriptomics'. All scripts, along with annotation, can be found [here](https://github.com/SizunJiangLab/CRS/tree/main/Scripts).

The full manuscript can be accessed [here](https://www.biorxiv.org/content/10.1101/2023.07.01.547352v1.abstract).

## Abstract

Chronic rhinosinusitis (CRS) is a common auto-inflammatory disease of the sinonasal cavity that affects millions of individuals worldwide. The complex pathophysiology of CRS remains poorly understood, with emerging evidence implicating the orchestration between diverse types of immune and epithelial cells in disease progression. We applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to both dissociated and intact human tissues to investigate the cellular and molecular heterogeneity of CRS with and without nasal polyp formation compared to non-CRS control samples. Our findings reveal a novel mechanism for macrophage-eosinophil recruitment into the nasal mucosa, dysregulation of CD4 and CD8 T cells, and an enrichment of mast cell populations to the upper airway tissues with intricate interactions between mast cells and CD4 T cells. We additionally identify key immune-epithelial interactions in sinonasal tissue remodeling, particularly involving understudied basal progenitor cells and tuft chemosensory cells. We further introduce a distinct basal cell differential trajectory implicated in NP formation through the immune-epithelial axis. Orthogonal validation of our findings was supported by spatial transcriptomics on >100 patients, with additional insights in immune-epithelium tissue remodeling events within CRSwNP. Using powerful clinical samples and advanced technologies, our study unveils novel aspects of the pathophysiology of CRS and sheds light on intricate immune and epithelial cell interactions within the disrupted CRS tissue microenvironment and promising targets for therapeutic intervention. These findings expand on existing knowledge of CRS and provide a comprehensive resource for understanding the cellular and molecular mechanisms underlying complex autoimmune diseases and beyond.

![Abstract](https://github.com/SizunJiangLab/CRS/blob/main/media/abstract.png)
