<p align="left">
  <img src="https://github.com/SizunJiangLab/CRS/blob/main/media/cropped-Jiangs-Logo.png" alt="Lab Logo" width="80">
</p>

# CRS
Code and scripts used in manuscript 'Immune-Epithelial Dynamics and Tissue Remodeling in Chronically Inflamed Nasal Epithelium via Multi-scaled Transcriptomics'. All scripts, along with annotation, can be found [here](https://github.com/SizunJiangLab/CRS/tree/main/Scripts).

The full manuscript can be accessed [here](https://www.biorxiv.org/content/10.1101/2023.07.01.547352v1.abstract).

The digital spatial profiling data have been deposited in Zenodo under [here](https://doi.org/10.5281/zenodo.17154922). The raw single-cell RNA-seq data are available in the GEO database under accession number GSE235715. All datasets are publicly accessible from the date of publication.

The website accompanying this manuscript can be accessed [here](https://bmblx.bmi.osumc.edu/immune_epi/) and via this backup [alternative](https://immune-epi.osubmi.org/)

## Abstract

Chronic rhinosinusitis (CRS) is a common chronic inflammatory disease of the sinonasal cavity that affects millions of individuals worldwide. The complex pathophysiology of CRS remains poorly understood, with emerging evidence implicating the orchestration between diverse types of immune and epithelial cells in disease progression, compared to non-CRS control samples}. We applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to both dissociated and intact human tissues to investigate the cellular and molecular heterogeneity of CRS with and without nasal polyp formation, compared to non-CRS control samples. Our findings suggest a potential mechanism for macrophage-eosinophil recruitment into the nasal mucosa, predict dysregulation of CD4 and CD8 T cells, and enrichment of mast cell populations in the upper airway tissues with intricate interactions with CD4 T cells. We additionally identify key immune-epithelial interactions in sinonasal tissue remodeling, particularly involving understudied basal progenitor cells and tuft chemosensory cells. We identify a distinct basal cell differential trajectory implicated in NP formation through the immune-epithelial axis. Orthogonal validation of our findings was supported by spatial transcriptomics on >100 patients, with additional insights into immune-epithelium tissue remodeling events within CRSwNP. Using powerful clinical samples and advanced technologies, our study provides new insights into the pathophysiology of CRS previously not known, highlighting intricate immune and epithelial cell interactions within the disrupted CRS tissue microenvironment as potential targets for therapeutic intervention. These findings build upon existing knowledge of tissue remodeling in chronic immunity to provide a detailed resource for understanding the cellular and molecular mechanisms underlying complex chronic immune diseases.

![Abstract](https://github.com/SizunJiangLab/CRS/blob/main/media/abstract.png)

## Contact

Please contact [Sizun Jiang](https://sizunjianglab.com/) at ```sjiang3@bidmc.harvard.edu``` if you have any questions about this repository. Please note the scripts here fall under an Academic usage license.

## Citation

```Liao, Guanrui, et al. "Immune-Epithelial Dynamics and Tissue Remodeling in Chronically Inflamed Nasal Epithelium via Multi-scaled Transcriptomics." bioRxiv (2023): 2023-07.```
